src/index.html

```diff
@@ -5,8 +5,349 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__links">
+          <li class="header__item">
+            <a href="#Apple" class="header__link is-active">Apple</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Samsung" class="header__link">Samsung</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Smartphones" class="header__link">Smartphones</a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#Laptops&Computers"
+              class="header__link"
+              data-qa="nav-hover"
+              >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Gadgets" class="header__link">Gadgets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Tablets" class="header__link">Tablets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Photo" class="header__link">Photo</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Video" class="header__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="card__imac"
+        >
+
+        <h1 class="card__imac-title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+        <p class="card__imac-code">Product code: 195434</p>
+
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+
+        <p class="card__review">Reviews: 5</p>
+
+        <p class="card__price">
+          Price:
+          <strong class="card__title-price">$2,199</strong>
+        </p>
+
+        <a
+          href="#buy"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div
+      class="card"
+      >
```

Comment:
```suggestion
      <div class="card">
```
if element have one attribute, and it's not long enough to put it on a new line, you can leave them on one line